### ISSUE TITLE
Release CM2LoD3 artifacts (models, dataset) on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [InFraHank/CM2LoD3#1]
Hi @InFraHank 🤗

Niels here from the open-source team at Hugging Face. I discovered your work on Arxiv (https://huggingface.co/papers/2508.15672) and was very interested in your CM2LoD3 method for reconstructing LoD3 building models. Your GitHub repository (https://github.com/InFraHank/CM2LoD3) indicates that the code is available, and you have developed pre-trained models (U-Net for conflict map processing, Mask R-CNN for window/door detection) and associated datasets (e.g., from the Semantic Conflict Map Generator).

I noticed your README currently states: "For access to pre-trained models and datasets, please contact the authors directly." It would be great to make these checkpoints and any associated datasets available on the 🤗 Hub, to improve their discoverability and visibility for the broader AI community. If you're planning a public release, hosting them on Hugging Face could provide a centralized and easily accessible platform.

We can help add relevant tags (e.g., `image-segmentation`, `object-detection` for models; `image-segmentation` for datasets) so that people can easily find your work when filtering `https://huggingface.co/models` and `https://huggingface.co/datasets`.

## Uploading models

You can find a guide here: https://huggingface.co/docs/hub/models-uploading.

In this case, you could leverage the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class, which adds `from_pretrained` and `push_to_hub` methods to any custom `nn.Module`. Alternatively, users can leverage the [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file) one-liner to download a checkpoint from the Hub.

We encourage researchers to push each model checkpoint to a separate model repository, so that features like download statistics also work. We can then also link the checkpoints to your paper page.

## Uploading dataset

It would be awesome to make your datasets (e.g., the SCMG-generated data or evaluation benchmarks) available on 🤗 , so that people can use:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/your-dataset")
```
You can find a guide here: https://huggingface.co/docs/datasets/loading.

Additionally, the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) allows people to quickly explore the first few rows of the data directly in the browser.

Let me know if you're interested in making these publicly available and if you need any guidance.

Cheers,

Niels
ML Engineer @ HF 🤗
